README.md

```diff
@@ -38,7 +38,6 @@ An example inventory can be found in [inventory/hosts.example](inventory/hosts.e
 Required variables include:
 
 - `openshift_kubeconfig_path` - Path to the kubeconfig for the cluster
-- `openshift_pull_secret_path` - Path to the pull secret to the image registry
```

Comment:
@kalexand-rh FYI, `openshift_pull_secret_path` is no longer a required input variable once this merges.

Reply:
https://bugzilla.redhat.com/show_bug.cgi?id=1712579